src/patternfly/components/Table/examples/Table.md

```diff
@@ -387,6 +387,7 @@ When header cells are empty or they contain interactive elements, `<th>` should
 | -- | -- | -- |
 | `.pf-c-table__check` | `<th>`, `<td>` | Initiates a check table cell. |
 | `.pf-c-table__action` | `<th>`, `<td>` | Initiates an action table cell. |
+| `.pf-c-table__inline-edit-action` | `<th>`, `<td>` | Initiates an inline edit action table cell. |
```

Comment:
should you add in that `pf-m-inline-editable`applies to `<tr>`

Reply:
actually does that do anything to the row?

Reply:
The row contains `.pf-c-inline-edit` componentry, so setting `.pf-m-inline-editable` on the row enables the editability of all its inline edit elements. The same is true for an entire form, wrapped in `.pf-c-inline-edit` or a single item (that could be in any context as its self contained). It can apply to any wrapping element or any `.pf-c-inline-edit` block. So it's not isolated to a `<tr>`, it can apply to almost anything.

Reply:
But that's not a function of table, it's a function of `.pf-c-inline-edit`, so it's documented in that component instead.

Reply:
got it, ok!